runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -141,6 +144,38 @@
     private String usernameFile;
     private String passwordFile;
 
+
+    /**
+     * Trigger the loading of all the SmrMaps upon connect.
+     *
+     * If using this utility, you need to be sure that no one
+     * is acessing objects until the tables are loaded
+     * (i.e. when connect return)
+     */
+    @Setter
+    @Getter
+    private boolean loadSmrMapsAtConnect = false;
+
+
+    /**
+     * Set the bulk read size for the Fast Laoder
+     */
+    @Setter
+    @Getter
+    private long bulkReadSizeForFastLoader = DEFAULT_BATCH_FOR_FAST_LOADER;
```

Comment:
Well, if I put them in the CorfuRuntimeParameters, i cannot chain them nicely with the corfuRuntime instantiation. If I can, I am not sure how. I searched for example of the usage of these corfuParameters I didn't really find any.